rustls/tests/common/mod.rs

```diff
@@ -218,6 +221,19 @@ impl KeyType {
             .collect()
     }
 
+    pub fn get_client_crl(&self) -> UnparsedCertRevocationList {
```

Comment:
Style nit: avoid `get_` prefixes.

Reply:
Done, but there are other instances of this anti-pattern I was matching. I will tidy those separately.